src/main/java/com/minecolonies/coremod/commands/ListColoniesCommand.java

```diff
@@ -102,7 +103,9 @@ public void execute(@NotNull final MinecraftServer server, @NotNull final IComma
 
         for (final Colony colony : coloniesPage)
         {
-            sender.addChatMessage(new TextComponentString(ID_TEXT + colony.getID() + NAME_TEXT + colony.getName()));
+            sender.addChatMessage(new TextComponentString(String.format(
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Format specifiers should be used instead of string concatenation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2275)